test/verify/check-superuser

```diff
@@ -121,11 +144,11 @@ session include system-auth
 """)
 
         b.open_superuser_dialog()
-        b.wait_in_text(".pf-c-modal-box:contains('Switch to administrative access')", "Password for admin:")
```

Comment:
Rebase error in the other direction, this belongs in the previous commit.

Reply:
Fixed them all, I think.